packages/@o3r/rules-engine/schemas/rulesets.schema.json

```diff
@@ -51,22 +51,25 @@
           "additionalItems": false
         },
         "linkedComponent": {
-          "type": "object",
+          "$ref": "#/definitions/LinkedComponent",
           "description": "Component linked to the ruleset, if set it will disable the ruleset execution per default, waiting to a subscription",
+          "deprecated": true
+        },
+        "linkedComponents": {
+          "type": "object",
+          "description": "Components linked to the ruleset. If defined, the rulest is not active by default",
           "properties": {
-            "library": {
-              "type": "string",
-              "description": "Name of the component library"
-            },
-            "name": {
-              "type": "string",
-              "description": "Name of the component"
+            "or": {
+              "type": "array",
+              "description": "List of components which will activate the ruleset. If at least one component from the list has subscribed, the ruleset will be active",
+              "items": {
+                "$ref": "#/definitions/LinkedComponent"
+              }
             }
           },
           "additionalItems": false,
           "required": [
-            "library",
-            "name"
+            "or"
```

Comment:
Should it be `required` if we want to be backward compatible at some point and introduce the `and` use case ?

Reply:
If I have 'linkedComponents' I want to have the 'or' always, but when adding the 'and' we'll have either one or the other :)) 
I think if we remove the required from the 'or' we stay backward compatible, no ?

Reply:
For our devtools I would agree, but we are also providing our metadata to CMS. If `or` is required in `linkedComponents` and we set it to optional later, we will introduce a breaking change.
We can make it optional directly I think. It should not matter on our side.

Reply:
indeed. Let's do that :)
This is a nice catch :) thanks